src/main/java/frc/robot/commands/defaultCommands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands.defaultCommands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends Command {
+
+    private final IntakeSubsystem intake;
+    private final DoubleSupplier bumper;
+
+    public IntakeDefaultCommand(IntakeSubsystem intake, DoubleSupplier bumper) {
+        this.bumper = bumper;
+        this.intake = intake;
+        addRequirements(intake);
+
+        //TODO:Remove after command has been tested
+        throw new UnsupportedOperationException("IntakeDefaultCommand has not been tested on robot yet!");
+    }
+
+    @Override
+    public void execute() {
+        intake.setIntakeSpeed(bumper.getAsDouble());
```

Comment:
Bumpers, unlike joysticks, only go from 0-1. Therefore, inputting one bumper means that you would be able to go one way. And we like to usually have it so that the right bumper is used to run the intake forwards and the left intake is used to make the intake run backwards. To fix this 2 fixes come to mind. The first would be to supply 2 double suppliers to the `IntakeDefaultCommand` one for the forward bumper and one for the backwards bumper, and check which one is currently being used and set the intake to that value. The other method is to pass one double supplier as you are currently doing, and when you instantiate `IntakeDefaultCommand`, pass in something like `rightBumper-leftBumper`, therefore you will combine the bumpers into a value that can go from -1 to 1.